notebooks/QA_extraction_generation.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<p>hahahah :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>